cmake/Tools/Platform/Android/android_support.py

```diff
@@ -1631,6 +1631,18 @@ def __init__(self, android_sdk_path, command_line_tools_version):
         self.available_updates = {}
         self.refresh_sdk_installation()
 
+    def call_sdk_manager(self, arguments, action):
+        result_code, result_stdout, result_stderr = self.sdk_manager.exec(arguments, capture_stdout=True, suppress_stderr=False)
+        if result_code != 0:
+            # Provide a more concise error if a known execption is thrown indicating the wrong version of JAVA is set in the environment
+            if "java.lang.UnsupportedClassVersionError" in result_stderr:
+                err_msg = "The current installed java runtime is not compatible with the sdkmanager. Make sure that " \
+                          "java runtime version 11 is installed and JAVA_HOME is set to that path."
```

Comment:
Java runtime version 11 is quite old.
Why would the error message specific that version?

Reply:
The compiled JAR files that have the sdkmanager and other class files were compiled with that version, so JDK8 was incompatible. The Android studio installation includes a local subfolder with its own java runtime ([jbr](https://developer.android.com/build/jdks#jdk-android-studio)), and is set in the studio.jdk config file, but when running from command line, it relies on %PATH% or %JAVA_HOME%.

Reply:
Is there a way to parameterized that version then?

Reply:
I don't think parameterizing the version is the right approach, nor is calling out a specific version. I'll look into the right way to determine what the expected version of java is required to run `sdkmanager.bat` and provide that message instead